app/models/document_type.rb

```diff
@@ -17,7 +17,9 @@ def self.all
       hashes = YAML.load_file(Rails.root.join("config/document_types.yml"))
 
       hashes.map do |hash|
-        hash["contents"] = hash["contents"].to_a.map(&Field.method(:new))
+        hash["contents"] = hash["contents"].to_a.map do |content|
```

Comment:
I think we should look at other ways of doing this, as it seems to be making the model more complicated. One way would be to add an equivalent method within the `Field` object. But since `label` is used so infrequently, can't we just do the translation in the view? Looking at [one of the tests](https://github.com/alphagov/content-publisher/blob/master/spec/features/editing_tags/edit_tags_spec.rb), it doesn't look like we're relying on it - if we are, we could think about using an ID instead.